app/Http/Controllers/CardDAV/CardDAVController.php

```diff
@@ -3,19 +3,22 @@
 namespace App\Http\Controllers\CardDAV;
```

Comment:
Shouldn't the name of this file `CardDAVController.php` be renamed to something else? Both Carddav and Caldav are from the WebDav protocol, so perhaps it would be a better name if we are to have one controller.